docs/guides/catalog.rst

```diff
@@ -406,7 +406,7 @@ Querying by Space and Time
 AttributeStore
 --------------
 
-When writing a layer GeoTrellis uses an ``AttributeStore`` to write layer metadata required to read and query the layer later.
+When writing a layer, GeoTrellis uses an :class:`~geopyspark.geotrellis.catalog.AttributeStore` to write layer metadata required to read and query the layer later.
```

Comment:
This should probably stay how it was since it's referring to GeoTrellis' `AttributeStore` and not GeoPySpark's

Reply:
Thats a good point. On reflection its probably better to simplify and talk about GeoPySpark, since that still true and more consistent.